src/datagrid/DataGridFilter.tsx

```diff
@@ -8,27 +8,28 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as React from "react";
-import {classNames} from "../utils";
+import React from "react";
+import {classNames, getXPositionValue} from "../utils";
+import {DEFAULT_TRANSFORM_XY_POSITIONS} from "../constants";
 import {ClassNames} from "./ClassNames";
 import {Button} from "../forms/button";
 import {DataGridRow} from "./DataGrid";
 import {DebounceUtils} from "../forms/common/DebounceUtils";
 
 /**
  * Props for DataGridFilter :
- * @param {style} CSS style
- * @param {className} CSS classnames
- * @param {datagridRef} Refrence for DataGrid on which filter will gets apply. We need this to call method which will update datagird rows.
- * @param {columnName} columnName on which filter will apply
- * @param {placeholder} placeholder for string filter input
- * @param {onFilter} Custom filter logic
- * @param {filterType} Type of filter string or custom
- * @param {disabled} boolean value to enable or disable filter
- * @param {debounce} boolean value to apply debounce behaviour
- * @param {debounceTime} number value debounceTime/Delay value in miliseconds
- * @param {position} position of the filter popup
- * @param {defaultValue} defaultValue of the filter
+ * @param {any} style - CSS style
+ * @param {string} className - CSS classnames
+ * @param {any} datagridRef - Reference for DataGrid on which filter will gets apply. We need this to call method which will update datagird rows.
+ * @param {string} columnName - columnName on which filter will apply
+ * @param {string} placeholder - placeholder for string filter input
+ * @param {Function} onFilter - Custom filter logic
+ * @param {FilterType} filterType - Type of filter string or custom
+ * @param {boolean} disabled - boolean value to enable or disable filter
+ * @param {boolean} debounce - boolean value to apply debounce behavior
+ * @param {number} debounceTime - number value debounceTime/Delay value in milliseconds
+ * @param {FilterPosition} position - position of the filter popup
+ * @param {any} defaultValue - defaultValue of the filter
```

Comment:
can u please re-check all comment changes? its showing datatype first and then prop name